doc/http-api/backend_swagger.yml

```diff
@@ -1,7 +1,7 @@
 swagger: '2.0'
 info:
   version: "1.0.0"
-  title: MongooseIM control backend REST API
+  title: "MongooseIM's backend REST API"
```

Comment:
We're not consistent: [normal documentation](http://mongooseim.readthedocs.io/en/latest/http-api/http-administration-api-documentation/) says _Administration_ API while in the [Swagger documentation](http://mongooseim.readthedocs.io/en/latest/swagger/index.html) we have _Control Backend_ API (which is too long a name anyway). We should change it. If no one disagrees by EOD then we'll make this change.